examples/demo11.py

```diff
@@ -209,10 +209,12 @@ def main(argv):
     theta = 0.17 * galsim.degrees
     # ( dudx  dudy ) = ( cos(theta)  -sin(theta) ) * pixel_scale
     # ( dvdx  dvdy )   ( sin(theta)   cos(theta) )
-    dudx = math.cos(theta.rad()) * pixel_scale
-    dudy = -math.sin(theta.rad()) * pixel_scale
-    dvdx = math.sin(theta.rad()) * pixel_scale
-    dvdy = math.cos(theta.rad()) * pixel_scale
+    # Aside: You can call numpy trig functions on Angle objects directly, rather than getting
```

Comment:
There is no difference.  

In fact, the numpy implementation for trig acting on objects looks for the corresponding method and calls that.  So `numpy.cos(theta)` just calls `theta.cos()` behind the scenes.  Doing the latter directly is (very) slightly more efficient, since it doesn't include the time for numpy to do that check.  Other than that, they are completely equivalent.